lambda-kinesis-lambda/README.md

```diff
@@ -0,0 +1,61 @@
+# This pattern creates 2 Lambda functions, one that puts data into the Kinesis Data Stream and another one that gets trigger when there is data in the stream. 
```

Comment:
This is good, but this is not the title of the file. This is a good intro to the file after the title. 

For the title, something like you had before is good.
# Serverless producer-consumer use case using AWS Lambda and AWS Kinesis Data Stream

Reply:
Done